composer.json

```diff
@@ -13,7 +13,7 @@
     "illuminate/database": "5.5.*",
     "illuminate/support": "5.5.*",
     "illuminate/queue": "5.5.*",
-    "enqueue/amqp-lib": "0.8.5",
```

Comment:
Let’s stick with 0.8.* for now. 0.9 has breaking changes.

Reply:
OK, I saw your comment somewhere about this.